app/controllers/waste_exemptions_engine/front_office_edit_forms_controller.rb

```diff
@@ -41,11 +41,13 @@ def validate_edit_token
     private
 
     def registration
-      @registration ||= Registration.where(edit_token: params[:edit_token]).first
+      @registration ||= Registration.where(edit_token: params[:edit_token].split("/")[0]).first
```

Comment:
Can edit tokens contain forward slashes? If so, that might break routes and navigation unless we're encoding it.